content/en/docs/collector/deployment-methods.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Deployment Methods
+weight: 2
+---
+
+The OpenTelemetry Collector consists of a single binary and two primary deployment methods:
```

Comment:
This page needs copyedits, but since it's just a refactoring of existing prose, I'll leave this as is for now.

Reply:
agreed, that whole page will need a followup PR to change some of the content.